### PR TITLE
Added : Navbar component

### PR DESCRIPTION
**Pull request by NeerajVodala**
Added:
- Navbar

Deploy Preview: https://deploy-preview-8--cluach.netlify.app/docs/components/navigation

Image for reference:

navbar

### CODE REVIEW

**Review by anshtripathi01 — APPROVED** [review]
UI Looks good and code is also clean and clear.

File: docs/components/navigation.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ckahub) [inline docs/components/navigation.html:7]: suggestion : u can use name=author and content = ur name

File: docs/components/navigation.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                        </div>
+                    </header>
+
+                </div>
+            </section>
+
+            <div class="code-snippet br-s">
+                <pre>
+                        <code class="html">
+&lt;header class="header"&gt;
```
Comment (ckahub) [inline docs/components/navigation.html:163]: give proper spacing here

**Review by ckahub — COMMENTED** [review]
code looks neat & clean

**Pull request merged**